src/ray/protobuf/core_worker.proto

```diff
@@ -231,4 +240,7 @@ service CoreWorkerService {
   rpc GetCoreWorkerStats(GetCoreWorkerStatsRequest) returns (GetCoreWorkerStatsReply);
   // Wait for a borrower to finish using an object. Sent by the object's owner.
   rpc WaitForRefRemoved(WaitForRefRemovedRequest) returns (WaitForRefRemovedReply);
+
+  // Plasma object ready.
```

Comment:
Can we be a little bit more descriptive? Something like Notify the core worker certain object id is ready in local plasma store.